src/Symfony/Component/HttpKernel/Profiler/ProfilerStorageInterface.php

```diff
@@ -21,16 +21,17 @@
     /**
      * Finds profiler tokens for the given criteria.
      *
-     * @param string   $ip     The IP
-     * @param string   $url    The URL
-     * @param string   $limit  The maximum number of tokens to return
-     * @param string   $method The request method
-     * @param int|null $start  The start date to search from
-     * @param int|null $end    The end date to search to
+     * @param string   $ip         The IP
+     * @param string   $url        The URL
+     * @param string   $limit      The maximum number of tokens to return
+     * @param string   $method     The request method
+     * @param int|null $start      The start date to search from
+     * @param int|null $end        The end date to search to
+     * @param int|null $statusCode The request status code
      *
      * @return array An array of tokens
      */
-    public function find($ip, $url, $limit, $method, $start = null, $end = null);
+    public function find($ip, $url, $limit, $method, $start = null, $end = null, $statusCode = null);
```

Comment:
adding new arguments in a method of an interface is a BC break, as it forces all implementation to add this parameter if they don't want to get a E_STRICT error